oap-server/server-storage-plugin/storage-banyandb-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/banyandb/BanyanDBNoneStreamDAO.java

```diff
@@ -48,7 +48,7 @@ public void insert(Model model, NoneStream noneStream) throws IOException {
         StreamWrite streamWrite = new StreamWrite(schema.getMetadata().getGroup(), // group name
                 model.getName(), // index-name
                 noneStream.id(), // identity
-                TimeBucket.getTimestamp(noneStream.getTimeBucket(), model.getDownsampling())); // timestamp
+                TimeBucket.getTimestamp(noneStream.getTimeBucket())); // timestamp
```

Comment:
The downsample value should be fixed, it is minute now

Reply:
Reverted back